changelog_entry.yaml

```diff
@@ -0,0 +1,4 @@
+- bump: patch
+  changes:
+    fixed:
+    - EndBug/add-and-commit updated to remove git commit warnings
```

Comment:
```suggestion
      - EndBug/add-and-commit updated to remove git commit warnings.
```

Reply:
Ah, that makes sense, thank you, and sorry for all of the hassle throughout.